### PR TITLE
Completed

### PR DESCRIPTION
**Pull request by Siri-isha**

### CODE REVIEW

File: BalancedTree.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+//T.C O(n)
+//S.C O(1)
```
Comment (ishika2741) [inline BalancedTree.java]: Check space complexity please

**Review by ishika2741 — COMMENTED** [review]
Good work!

**Comment by ishika2741** [comment]
Good work!

File: BalancedTree.java (hunk 1 above)
Comment (Siri-isha) [reply BalancedTree.java]: > Check space complexity please

Please review updated space complexity

**Pull request closed without merging**